Dapper/CommandFlags.cs

```diff
@@ -24,5 +24,9 @@ public enum CommandFlags
         /// Should the plan cache be bypassed?
         /// </summary>
         NoCache = 4,
+        /// <summary>
+        /// Should the cancellation be allowed after result collection starts?
+        /// </summary>
+        RegisterCancellation = 5,

```

Comment:
this would be `8`, not `5` - it is a flags enum; alternatively: `Buffered = 1 <<  0`, `Pipelined = 1 << 1`, `NoCache = 1 << 2`, `RegisterCancellation = 1 << 3`